### PR TITLE
[Sprint 1] Profile detail

### PR DESCRIPTION
**Pull request by NguyenTienVinh**
ProfileDetailFragment

Link redmine: https://dev.framgia.com/redmine/issues/112030
Task: Load detail profile user
Affect: Update user profile

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -32,6 +32,10 @@
...
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme"/>
+        <activity
+            android:name=".screen.profilepage.profiledetail.ProfileDetailActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/AppTheme"/>
     </application>
 
-</manifest>
+</manifest>
```
Comment (daolq3012) [inline app/src/main/AndroidManifest.xml]: add enter when end of file

File: app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java [hunk 2]
```diff
@@ -56,8 +57,8 @@ public String getEmail() {
         return mUser.getEmail();
     }
 
-    public void onClickUpdateProfile() {
-        //TODO: open Update Profile
+    public void onClickProfileDetail() {
+        mNavigator.startActivity(ProfileDetailActivity.class);
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java]: ProfileDetail is a Fragment please change!

File: app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java [hunk 3]
```diff
@@ -56,8 +57,10 @@ public String getEmail() {
         return mUser.getEmail();
     }
 
-    public void onClickUpdateProfile() {
-        //TODO: open Update Profile
+    public void onClickProfileDetail() {
+        mNavigator.goNextChildFragment(R.id.layout_content,
+                ProfileDetailFragment.newInstance(), true, Navigator.NONE,
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java]: Have you try Animation RIGHT_LEFT ?

File: app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java [hunk 4]
```diff
@@ -56,8 +57,10 @@ public String getEmail() {
...
+                "ProfileDetailFragment");
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java]: Please declare on top of file
```java
private static final String TAG = "ProfileDetailFragment";
```
And use it for tag of fragment when commit

File: app/src/main/res/layout/fragment_profile_detail.xml [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+                            android:layout_marginBottom="@dimen/dp_3"
+                            android:layout_marginTop="@dimen/dp_3"
+                            android:textSize="@dimen/sp_16"
+                            />
+                    </LinearLayout>
+                </LinearLayout>
+            </LinearLayout>
+        </LinearLayout>
+    </ScrollView>
+</layout>
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_profile_detail.xml]: add a empty line when end of file

File: app/src/main/res/layout/fragment_profile_detail.xml [hunk 6]
```diff
@@ -0,0 +1,375 @@
+<?xml version="1.0" encoding="utf-8"?>
+
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_profile_detail.xml]: remove empty line

File: app/src/main/res/layout/fragment_profile_detail.xml [hunk 7]
```diff
@@ -0,0 +1,375 @@
...
+    <data>
+
+        <variable
+            name="viewModel"
+            type="com.framgia.forder.screen.profilepage.profiledetail.ProfileDetailViewModel"
+            />
+
+    </data>
+
+
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_profile_detail.xml]: remove empty line

File: app/src/main/java/com/framgia/forder/screen/profilepage/profiledetail/ProfileDetailViewModel.java [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    @Override
+    public void onGetProfileDetail(User user) {
+        mUser = user;
+    }
+
+    public String getName() {
+        if (mUser == null) {
+            return "";
+        }
+        return mUser.getName();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/forder/screen/profilepage/profiledetail/ProfileDetailViewModel.java]: return mUser !=null? mUser.getName():"";

File: app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java [hunk 9]
```diff
@@ -3,14 +3,15 @@
...
+import com.framgia.forder.screen.profilepage.profiledetail.ProfileDetailFragment;
 import com.framgia.forder.utils.navigator.Navigator;
 
 /**
  * Exposes the data to be used in the ProfileFragment screen.
  */
 
 public class ProfilePageViewModel implements ProfilePageContract.ViewModel {
-
+    private static final String TAG = "ProfileDetailFragment";
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/forder/screen/profilepage/ProfilePageViewModel.java]: please add 1 empty line bellow this line

**Pull request merged**